doc/march_packages/march_input_device.rst

```diff
@@ -29,6 +29,24 @@ Receiving messages
 Tutorials
 ---------
 
+Set correct ubuntu permissions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The following error can occur when trying to access the ttyUSB port:
+
+.. code::
+
+   Cannot open /dev/ttyUSB0: Permission denied
+
+To make sure your user is allowed to access the port, add your user to the required groups:
+
+.. code::
+
+  sudo usermod -a -G tty <your_username>
```

Comment:
Is this your github username or the username of the computer? Might be useful to mention

Reply:
The name of your ubuntu user

Reply:
I don't think there should be any confusion as to which user this refers to.